cas/service.c

```diff
@@ -1330,6 +1333,14 @@ static int cas_fom_tick(struct m0_fom *fom0)
 		 * fixed for the correct credit calculation.
 		 */
 		break;
+	case CAS_TXN_OPENED:
+		m0_fom_phase_set(fom0, is_meta ? CAS_LOOP : CAS_META_UNLOCK);
+		break;
+	case CAS_META_UNLOCK:
+		M0_ASSERT(!is_meta);
+		m0_long_read_unlock(m0_ctg_lock(meta), &fom->cf_meta);
```

Comment:
For meta, when does unlock happen?

Reply:
For meta operations _fom->cf_ctg_ points to meta catalogue, it's released when operation is done.